docs/docs/development/hardware-integration/new-shield.mdx

```diff
@@ -81,13 +81,14 @@ Follow these steps to create your new repository:
 - Click the green "Create repository" button
 
 The repository is a combination of the directories and files required of a ZMK config, and those required of a shield module.
-To create a shield module, the following components are needed:
+This enables the use of GitHub Actions to test that the shield is defined correctly.
+See also the page on [module creation](../module-creation.md) for information on exactly which file structure and files are required for a ZMK keyboard module.
```

Comment:
```suggestion
See also the page on [module creation](../module-creation.md) for a reference on exactly which file structure and files are required for a ZMK keyboard module.
```
Tiny change, but trying to convey that reading the page is optional, for the purposes of following this guide.